plugins/arcgis/service/src/ObservationProcessor.ts

```diff
@@ -120,8 +120,13 @@ export class ObservationProcessor {
      * Gets the current configuration from the database.
      * @returns The current configuration from the database.
      */
-    public async safeGetConfig(): Promise<ArcGISPluginConfig> {
-        return await this._stateRepo.get().then(x => !!x ? x : this._stateRepo.put(defaultArcGISPluginConfig))
+    public async safeGetConfig(showFeatureAuth?: boolean): Promise<ArcGISPluginConfig> {
```

Comment:
Sanitization should happen in the context of web responses, web routes can handle this.

Reply:
done